Tests/Unit/Hooks/PageRenderer/TrackingCodeInjectorTest.php

```diff
@@ -258,7 +287,8 @@ public function executeAddsJavaScriptTrackingCodeWithTagManagerToHeaderDataCorre
             $this->requestStub,
             $this->javaScriptTrackingCodeBuilderStub,
             $this->noScriptTrackingCodeBuilderStub,
-            $this->tagManagerCodeBuilderStub
+            $this->tagManagerCodeBuilderStub,
+            $this->scriptTagBuilder
```

Comment:
Hmm, I am undecided to use the concrete implementation. Can be changed later perhaps.